sound/soc/sof/sof-ipc-test-client.c

```diff
@@ -6,22 +6,268 @@
 //
 
 #include <linux/completion.h>
+#include <linux/debugfs.h>
```

Comment:
can we squash this one? fixup! is usually for stuff that's already merged.

Reply:
yes, done